schema-guide.md

```diff
@@ -581,12 +581,15 @@ such as *`definitions.alias`*.
 
 - **type**: Nonempty `string`
 - **required**: `false`
-- **description**: A date. Format is 4-digit year, 2-digit month, 2-digit day of month, separated by dashes.
+- **description**: A date. Format is 4-digit year, 2-digit month, 2-digit day of month, separated by dashes, and double-quoted or unquoted.
```

Comment:
Would it be worth to mention that the date is in ISO 8601 format?

Reply:
To be honest, I don't think that this would add anything helpful for users, especially because we define a strict subset of ISO 8601.

Reply:
And thanks for the review :)

Reply:
maybe we can still add something like
```suggestion
- **description**: A date. Format is 4-digit year, followed by 2-digit month, followed by 2-digit day of month, separated by dashes, and double-quoted or unquoted.
```